src/Components/NavBar/NavBar.css

```diff
@@ -12,3 +13,20 @@
   font-weight: 900;
   transform: scale(1.1);
 }
```

Comment:
I just noticed that the active NavBar buttons were only a different color because of the hover property. Once you click away from it, it becomes a little hard to tell the difference between the buttons. Maybe we can also add the background-color to the `nav-selected` class?
![Screen Shot 2020-11-27 at 10 23 05 AM](https://user-images.githubusercontent.com/39573970/100476500-95b9a300-309a-11eb-8139-3c3409c1c8c2.png)
![Screen Shot 2020-11-27 at 10 23 13 AM](https://user-images.githubusercontent.com/39573970/100476502-96523980-309a-11eb-8728-8af3e511c16b.png)